### PR TITLE
Lint/prettier

### PR DESCRIPTION
**Pull request by 1kh4l**
Add prettier and some rules for better code formatting

### CODE REVIEW

File: src/components/TheSideNavMenu.vue [hunk 1]
```diff
@@ -4,14 +4,14 @@
     <!-- LIST ITEMS CONTAINER-->
     <v-list dense nav>
       <!-- HOME -->
-      <router-link to= '/home' v-slot="{href, route, navigate, isActive, isExactActive }">
-      <!-- Creates home menu without subitem -->
-        <v-list-item  @click="menuRouting({ title: 'home' })">
+      <router-link to="/home" v-slot="{}">
```
Comment (1kh4l) [inline src/components/TheSideNavMenu.vue:7]: what happens if you delete v-slot?

Comment (agderes) [reply src/components/TheSideNavMenu.vue:7]: we dont want underlines :)

File: src/plugins/i18n.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import Vue from 'vue';
-import VueI18n from 'vue-i18n';
+import Vue from "vue";
+import VueI18n from "vue-i18n";
 
 Vue.use(VueI18n);
 
-function loadLocaleMessages() {
-  const locales = require.context('./../locales', true, /[A-Za-z0-9-_,\s]+\.json$/i);
+const loadLocaleMessages = function () {
```
Comment (1kh4l) [inline src/plugins/i18n.js:6]: what if we do 
`const loadLocaleMessages = () => {`

Comment (agderes) [reply src/plugins/i18n.js]: It's changed!

**Pull request merged**